.github/workflows/push.yml

```diff
@@ -545,6 +562,9 @@ jobs:
       - name: Retag unaffected Docker images
         if: "!startsWith(env.GIT_BRANCH_DEPLOY, 'feature/')"
         run: ./scripts/ci/retag-unaffected.sh "$UNAFFECTED"
+      - name: Retag unaffected Docker images with lifecycle
+        if: "!startsWith(env.GIT_BRANCH_DEPLOY, 'feature/')"
+        run: ./scripts/ci/retag-lifecycle-policy.sh "$UNAFFECTED" #Retags feature images for lifecycle. 
```

Comment:
Can you do something like `DOCKER_TAG=feature-$POLICY_TAG ./scripts/ci/retag-unaffected.sh "$UNAFFECTED"` instead of creating the additional retag script?

Reply:
That's actually something I meant to do after reading over my code. 
I'll make the change